src/app/forms/suggestion-box/terra-suggestion-box.component.ts

```diff
@@ -258,6 +247,8 @@ export class TerraSuggestionBoxComponent implements OnInit, OnChanges
         let searchString = this._selectedValue.caption;
         this.toggleOpen = true;
 
+        this.updateSelectedSuggestionBoxEntryBySearchString(searchString);
```

Comment:
Hier änderst du eine grundlegende Funktionalität der Komponente. Hast du geprüft ob überall in Terra die SuggestionBox einen beliebigen String zulassen darf?